.gitignore

```diff
@@ -36,3 +36,5 @@ workload/.python-version
 # notebook outputs
 docs/harmony_dimension_subset_results/
 docs/outputs
+
+.netrc
```

Comment:
I assume this isn't needed with the latest changes (I don't have a .netrc file in my directory after running the notebooks).